Blazor/wwwroot/cdts-blazor.js

```diff
@@ -0,0 +1,234 @@
+﻿//Get and set the culture name in the local storage
+window.blazorCulture = {
+    get: () => window.localStorage['BlazorCulture'],
+    set: (value) => window.localStorage['BlazorCulture'] = value
+};
+
+var cdtsBlazor = {
+    exitScriptObj: null,
+    globalSerializedFooter: null,
+    globalSerializedTop: null,
+    globalSerializedPreFooter: null,
+    globalSerializedSectionMenu: null,
+    currentPage: null,
+    cdnEnv: null, 
+    defaultCSSHref: "https://www.canada.ca/etc/designs/canada/cdts/gcweb/v5_0_0/cdts/cdts-styles.css",
+
+    //Functions
+    setRefTop: function setRefTop(serializedSetupBase, exitSecureSiteObj, isApp) {
```

Comment:
As mention in previous, comment, `exitSecureSiteObj` can be taken from `serializedSetupBase` **AND** it might not be necessary to deserialize `serializedSetupBase`, we might be able to take directly `setupBase`

Reply:
done